app/src/main/java/com/kickstarter/libs/Either.kt

```diff
@@ -0,0 +1,61 @@
+package com.kickstarter.libs
+
+sealed class Either<out A, out B> {
```

Comment:
for posterity `public` is the default modifier in Kotlin so we get redundant warnings